bootloader/src/pyi_python.c

```diff
@@ -122,12 +134,15 @@ pyi_python_map_names(HMODULE dll, int pyvers)
     GETPROC(dll, PyErr_Occurred);
     GETPROC(dll, PyErr_Print);
     GETPROC(dll, PyErr_Fetch);
+    GETPROC(dll, PyErr_GivenExceptionMatches);
+    GETPROC(dll, Py_Exit);
     GETPROC(dll, PyImport_AddModule);
     GETPROC(dll, PyImport_ExecCodeModule);
     GETPROC(dll, PyImport_ImportModule);
     GETPROC(dll, PyList_Append);
     GETPROC(dll, PyList_New);
     GETPROC(dll, PyLong_AsLong);
+    // GETPROC(dll, PyLong_Check);
     GETPROC(dll, PyModule_GetDict);
```

Comment:
If I add this in, the bootloader doesn't run. Any ideas? Same for the other commented out line above.